deployment/amplify/production/amplify/#current-cloud-backend/auth/echolocatorCognitoProduction/parameters.json

```diff
@@ -0,0 +1,66 @@
+{
+    "identityPoolName": "echolocatorIdentityPoolProduction",
+    "allowUnauthenticatedIdentities": false,
+    "thirdPartyAuth": false,
+    "lambdaLogPolicy": "echolocatorCognitoProduction_lambda_log_policy",
+    "openIdLambdaRoleName": "echolocatorCognitoProduction_openid_lambda_role",
+    "openIdRolePolicy": "echolocatorCognitoProduction_openid_pass_role_policy",
+    "openIdLambdaIAMPolicy": "echolocatorCognitoProduction_openid_lambda_iam_policy",
+    "openIdLogPolicy": "echolocatorCognitoProduction_openid_lambda_log_policy",
+    "userPoolName": "echolocatorUserPoolProduction",
+    "autoVerifiedAttributes": [
+        "email"
+    ],
+    "mfaConfiguration": "OFF",
+    "mfaTypes": [
+        "SMS Text Message"
+    ],
+    "roleName": "echolocatorCognitoProduction_sns-role",
+    "roleExternalId": "echolocatorCognitoProduction_role_external_id",
+    "policyName": "echolocatorCognitoProduction-sns-policy",
+    "smsAuthenticationMessage": "Your authentication code is {####}",
+    "smsVerificationMessage": "Your verification code is {####}",
+    "emailVerificationSubject": "Your verification code",
+    "emailVerificationMessage": "Your verification code is {####}",
+    "defaultPasswordPolicy": false,
+    "passwordPolicyMinLength": 8,
+    "passwordPolicyCharacters": [
+        "Requires Lowercase",
+        "Requires Uppercase",
+        "Requires Numbers",
+        "Requires Symbols"
+    ],
+    "requiredAttributes": [
+        "email"
+    ],
+    "userpoolClientName": "echolocatorCognitoProduction_app_client",
+    "userpoolClientGenerateSecret": true,
+    "userpoolClientRefreshTokenValidity": "90",
+    "userpoolClientReadAttributes": [
+        "email",
+        "phone_number",
+        "updated_at",
+        "email_verified",
+        "phone_number_verified"
+    ],
+    "mfaLambdaRole": "echolocatorCognitoProduction_totp_lambda_role",
+    "mfaLambdaLogPolicy": "echolocatorCognitoProduction_totp_lambda_log_policy",
+    "mfaPassRolePolicy": "echolocatorCognitoProduction_totp_pass_role_policy",
+    "mfaLambdaIAMPolicy": "echolocatorCognitoProduction_totp_lambda_iam_policy",
+    "userpoolClientLambdaRole": "echolocatorCognitoProduction_userpoolclient_lambda_role",
+    "userpoolClientLogPolicy": "echolocatorCognitoProduction_userpoolclient_lambda_log_policy",
+    "userpoolClientLambdaPolicy": "echolocatorCognitoProduction_userpoolclient_lambda_iam_policy",
+    "userpoolClientSetAttributes": true,
+    "useDefault": "manual",
+    "authSelections": "identityPoolAndUserPool",
+    "resourceName": "echolocatorCognitoProduction",
+    "userpoolClientWriteAttributes": [
```

Comment:
These were defined in the previous production configuration, but undefined in the staging configuration.

Reply:
Good catch! Do you think it's worth aligning this with the staging stack?

Reply:
No, staging is working fine, so I think it would be better to leave it. If the issue should appear on staging later, we can revisit it then.